templates/resources/central-configuration/help.md.j2

```diff
@@ -0,0 +1,41 @@
+% {{ product | upper }} (1) Container Image Pages
+% Digital.ai Development Team
+% {{ today }}
+
+# NAME
+{{ product }} \- {{ product_description }}
+
+# DESCRIPTION
+{% include 'includes/' + product + '-description.j2' %}
+
+# ENVIRONMENT VARIABLES
+The following environment variables can be set to configure the image for different scenarios
+
+APP_ROOT=/opt/xebialabs
+    This environment variable is used to determine the directory where all application specific code will be copied. Normally there is no need to change this variable.
+
+APP_HOME=/${APP_ROOT}/{{ product }}-server
+    This environment variable is used as the landing point for all the application data. Ensure that this is a subdirectory of the ${APP_ROOT} variable. Normally there is no need to change this variable.
+
+# VOLUMES
+The following locations can be mounted as volumes to provide either configuration data, and/or persistent storage for application data.
+
```

Comment:
we need to add centralConfiguration volume also

Reply:
corrected the docs